src/main/java/seedu/address/ui/StatsWindow.java

```diff
@@ -44,6 +44,10 @@ public StatsWindow() {
         this(new Stage());
     }
 
+    public void setPieChart(ObservableList<PieChart.Data> newData) {
```

Comment:
Can consider adding javadocs for this method too!

Reply:
ok will do!